event-log/src/main/scala/io/renku/eventlog/Microservice.scala

```diff
@@ -93,47 +94,56 @@ object Microservice extends IOMicroservice {
                                                        ApplicationLogger
                                 )
         subscribers <- Subscribers(ApplicationLogger)
-        eventsDistributor <- EventsDistributor(transactor,
-                                               subscribers,
-                                               waitingEventsGauge,
-                                               underProcessingGauge,
-                                               queriesExecTimes,
-                                               ApplicationLogger
-                             )
-        subscriptionsEndpoint <- IOSubscriptionsEndpoint(subscribers, ApplicationLogger)
-        microserviceRoutes = new MicroserviceRoutes[IO](
-                               eventCreationEndpoint,
-                               latestEventsEndpoint,
-                               processingStatusEndpoint,
-                               eventsPatchingEndpoint,
-                               statusChangeEndpoint,
-                               subscriptionsEndpoint,
-                               new RoutesMetrics[IO](metricsRegistry)
+        eventFetcher <-
```

Comment:
I believe the fetcher and distributor won't be instantiated here in the future but maybe in the apply of the registry?

Reply:
Yes exactly this will all be hidden